README.md

```diff
@@ -33,12 +33,12 @@ If no puzzle file is specified, `puzzle_files/example1.txt` is used by default.
 There are three constraints used in formulating this problem:
 1. Minimize the difference between squares of left-hand side and right-hand side of equation (minimum of zero).
 2. Assignments of digits to letters must be unique, thus a puzzle may contain no more than 10 unique letters.
-3. The first letter must be non-zero, unless the puzzle contains one letter long components.
+3. The first letters must be non-zero, unless the puzzle contains one letter long components.
```

Comment:
Do you mean "The first letter of each word must be assigned a non-zero digit"?
Also "contains one-letter-long components" or better yet "contains one-letter words"